spec.emu

```diff
@@ -484,6 +484,81 @@
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-position-types">
+  <h1>Position types</h1>
+
+  <emu-clause id="sec-position-record-type">
+    <h1>Position Record</h1>
+    <p>A <dfn>Position Record</dfn> is a tuple of a non-negative line and non-negative column number:</p>
+    <emu-table id="table-position-record-fields" caption="Position Record Fields">
+      <table>
+        <thead>
+          <tr>
+            <th>Field Name</th>
+            <th>Value Type</th>
+          </tr>
+        </thead>
+        <tr>
+          <td>[[Line]]</td>
+          <td>a Number</td>
```

Comment:
Just for clarity, what do you think about putting "a non-negative integral Number" here and below?

Reply:
Good point, done!